x/concentrated-liquidity/tick_test.go

```diff
@@ -1405,89 +1405,3 @@ func (s *KeeperTestSuite) TestValidateTickRangeIsValid() {
 		})
 	}
 }
-
-func (s *KeeperTestSuite) TestRoundTickToCanonicalPriceTick() {
```

Comment:
Why is this removed? Is there a replacement for these test vectors?

Reply:
The notion of canonical price ticks no longer exist, so these vectors no longer make sense. All the test vectors added relating to the behavior in ticks surrounding the new MinTick ensure the appropriate properties hold.